microprofile-metrics/README.adoc

```diff
@@ -85,19 +99,19 @@ Now we need to add the following dependencies:
 
 [source,xml]
 ----
-<!-- Import the MicroProfile Metrics API, we use provided scope as the API is included in WildFly -->
+<!-- Import the MicroProfile Metrics API, we use provided scope as the API is included in the server -->
```

Comment:
Will these also be updated in pom.xml?

Reply:
missed this, fixed by https://github.com/wildfly/quickstart/pull/419